recipe/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "kaldi-native-fbank" %}
+{% set version = "1.17" %}
+
+# version is in format 1.X_Y_ZZZZZZ because the release is based on 1.17 from the main developer
+# but a commit was added from a fork from developer cbalioglu
+package:
+  name: {{ name|lower }}
+  version: {{ version }}+1_740f131
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b877a6aea7b09f87dfefd9e01106f38221287a2369fcda9221beb6bb76476d5c
+  patches:
+    - 0001-unvendor-gtest-and-pybind11.patch
+    - 0002-install-cmake-config-for-package.patch
+    - 0003-Update-ExtractWindow-signature.patch
+
+build:
+  # not a must-have, there is a problem with cmake -GNinka not picking up ninja
+  skip: True  # [win]
+  number: 0
+  ignore_run_exports:
+    - gtest
+  missing_dso_whitelist:
+    - $RPATH/ld64.so.*  # [s390x]
+    - $RPATH/libkaldi-native-fbank-core.dylib  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+    - make # [not win]
+    - ninja
+    - git  # [not win]
+    - patch  # [not win]
+    - m2-patch  # [win]
+  host:
+    - python
+    - pip
+    - setuptools
+    - wheel
+    - pybind11 2.12.0
+    - gtest 1.14.0
+  run:
+    - python
+    # part of the targets that are build are test-log, test-online-fbank and test-rfft which link to gtest libraries
+    - gtest
```

Comment:
gtest should never be added to the runtime deps.

Reply:
how do you cope with the binaries requiring it? the only way to test this is producing those binaries as part of the build with `cmake`. Generating those targets in the test section would prove pretty challenging without having to patch the `CMakeLists.txt` heavily.

Reply:
You run the tests during the build. That's the only sane way to do it today unfortunately.

Reply:
you mean a la `cmake --build . --target test` or something?

Reply:
Yes. You can also call ctest I believe.

Reply:
ctest did not work (not sure if it's because upstream `CMakeLists.txt` was not properly configured.
Anyway, now the individual test targets are called after `cmake` is done running.

Reply:
Yep, that's fine 👍